<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeworkWindow</class>
 <widget class="QMainWindow" name="HomeworkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>41</y>
      <width>1031</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_subject">
       <property name="text">
        <string>Przedmiot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_teacher">
       <property name="text">
        <string>Nauczyciel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_order">
       <property name="text">
        <string>Polecenie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_pubDate">
       <property name="text">
        <string>Data udostępnienia zadania</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_numberOfStudents">
       <property name="text">
        <string>Ilość uczniów</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_numberOfCompletedTasks">
       <property name="text">
        <string>Ilość odesłanych prac</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_endDate">
       <property name="text">
        <string>Termin wykonania zadania</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Akcja</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
